commit 14aa584f6d005b787b096ab219055f12a41469bd (refs/changes/14/4614/3)
Author: Jason King <jason.king@joyent.com>
Date:   2018-08-01T23:42:31+00:00 (1 year, 2 months ago)
    
    OS-7100 Build fails in libvscan after merge
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/lib/libvscan/Makefile b/usr/src/lib/libvscan/Makefile
index 1cf8f5effa..1212cc6a73 100644
--- a/usr/src/lib/libvscan/Makefile
+++ b/usr/src/lib/libvscan/Makefile
@@ -42,7 +42,7 @@ MSGFILES =	common/libvscan.c
 
 .KEEP_STATE:
 
-all clean clobber install lint _msg : $(SUBDIRS)
+all clean clobber install lint: $(SUBDIRS)
 
 $(POFILE):	$(MSGFILES)
 	$(BUILDPO.msgfiles)
